<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumView</class>
 <widget class="QWidget" name="SpectrumView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>373</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>This page gives an overview of the spectrum of this element. Using the mouse you can zoom into the spectrum.</string>
  </property>
  <property name="whatsThis">
   <string>This page gives an overview of the spectrum of this element.</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label2">
           <property name="text">
            <string>Maximum &amp;value:</string>
           </property>
           <property name="buddy">
            <cstring>maximumValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maximumValue">
           <property name="toolTip">
            <string>This sets the rightmost wavelength of the spectrum.</string>
           </property>
           <property name="whatsThis">
            <string>This sets the rightmost wavelength of the spectrum.</string>
           </property>
           <property name="suffix">
            <string>nm</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>40000</number>
           </property>
           <property name="value">
            <number>780</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label1">
           <property name="text">
            <string>&amp;Minimum value:</string>
           </property>
           <property name="buddy">
            <cstring>minimumValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="minimumValue">
           <property name="toolTip">
            <string>This sets the leftmost wavelength of the spectrum.</string>
           </property>
           <property name="whatsThis">
            <string>This sets the leftmost wavelength of the spectrum.</string>
           </property>
           <property name="suffix">
            <string>nm</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>40000</number>
           </property>
           <property name="value">
            <number>380</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Selected Peak</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Intensity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="intensity_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Wavelength:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="wavelength_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>List of all Peaks</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="peakList">
         <column>
          <property name="text">
           <string>Number</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Wavelength</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Intensity</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="SpectrumWidget" name="m_spectrumWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>This is the spectrum of the element.</string>
     </property>
     <property name="whatsThis">
      <string>This is the spectrum of the element.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectrumWidget</class>
   <extends>QWidget</extends>
   <header>spectrumwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
